docs/docs/reference/other-new-features/explicit-nulls.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: doc-page
+title: "Explicit Nulls"
+---
+
+This proposal describes a modification to the Scala type system that makes reference types
```

Comment:
If it's merged it's no longer a proposal, so this needs another introduction.

Reply:
Generally, this should be slightly revised so that it'a doc page for a Scala developer. This means providing more examples and simple explanations. There should be another page with more details that is linked to at the end of the doc page. I would defer all soundness discussions to this second page. 

So the first page should be and example driven use-level description, and the second page can then fill in spec, meta-properties such as soundness, and discussion of design tradeoffs.

The docs should also cover flow analysis and how flow analysis links up with typing. See my comment https://github.com/lampepfl/dotty/pull/6344#issuecomment-554062532.